internal/jobtracker/jobtracker_test.go

```diff
@@ -105,6 +105,21 @@ func (s *jobTrackerSuite) TestRun_JobSetSuccessful(c *qt.C) {
 	s.pollJob(testCtx, id, c, dbmodel.StatusSuccessful)
 }
 
+func (s *jobTrackerSuite) TestRun_JobIdSetInContext(c *qt.C) {
+	testCtx := c.Context()
+
+	aJobWithIdInContext := func(ctx context.Context) error {
+		_, ok := jobtracker.JobIdFromContext(ctx)
```

Comment:
Minor but I suggest to check that the value is non-zero or something along those lines.